### ISSUE TITLE
Unable to run Example Data

### ISSUE THREAD

**Issue by DanielRichardBiber** [KHP-Informatics/ADR-graph#2]
I have downloaded the data and docker image.  The initiation of the EdgePrediction image throws an error.  This is from the second line in the ExampleUseDoc.rst.

`ep = EdgePrediction.EdgePrediction()
`
AttributeError                            Traceback (most recent call last)
<ipython-input-3-729e57545ade> in <module>
----> 1 ep = EdgePrediction.EdgePrediction()

AttributeError: class EdgePrediction has no attribute 'EdgePrediction'

Even running the following 
`ep = EdgePrediction()
`
I run into other errors later.

Thank you for your time in developing this, and I do recognize that this hasn't seen a lot of attention but despite that, I would really like to see this package work.

**Comment by dbeanm** [comment]
Hi, yes you're right the ExampleUseDoc is out of date, I've just updated it. Thanks for pointing that out. The example is the code from test_library.py - I just cloned the repo onto a new machine to test and that does work for me. So it looks like just an issue with the docs. Does running test_library.py give you any errors?

**Comment by DanielRichardBiber** [comment]
I was able to run test_library.py successfully without errors this morning.  I'll be honest switching from some of the lines in ExampleUseDoc to test_library was a bit confusing.  Thank you for updating, I really appreciate it!

You might consider taking the "as ep" part of loading the EdgePrediction class out of the ExampleUseDoc (or call your EdgePrediction object something other than "ep") for ease of running.  I noticed things were a bit cleaner in the test_library.py doc.
